assets/js/data/store-notices/reducers.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import type { Reducer } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { defaultStoreNoticesState, StoreNoticesState } from './default-state';
+
+const reducer: Reducer< StoreNoticesState > = (
+	state = defaultStoreNoticesState,
+	action
+) => {
+	switch ( action.type ) {
+		case 'REGISTER_CONTAINER':
```

Comment:
Would it be better to add these constants to a `action-types.ts` file? That's the pattern we used in other data stores.